doc/languages-frameworks/julia.section.md

```diff
@@ -0,0 +1,23 @@
+# Julia {#sec-julia}
+
+### Installing Julia Packages {#sssec-installing-julia-packages}
+
+Most Julia compilers available in Nixpkgs have a passthru attribute
```

Comment:
Shouldn't all have that?

Reply:
When I wrote this, one compiler was broken. It has now been fixed and I've added the `passthru`. I'll update the sentence.